### DISCUSSION TITLE
Issue where the generated field of my output type is not the same as my POKO entity?

### DISCUSSION THREAD

**Discussion by wojcik-marcin** [leangen/graphql-spqr#461] [Q&A]
I would like to request some guidance with an issue I am experiencing.

I am making use of the SPQR Spring Boot Starter v0.0.7 within my Kotlin Spring Boot application.

I have a Person data class defined as follows:

``` kotlin
data class Person(
   val name: String,
   val isMarried: Boolean
)
```

I have a @GraphQLMutation endpoint to update the Person

 ``` kotlin
@GraphQLMutation
fun updatePerson(@GraphQLArgument(name = "person") person: Person): Person {
    person.isMarried = true
    return Person
 }
 ```
 
 The GraphQL schema types generated are as follows:
 
 ``` graphql 
type Person {
    name: String!
    married: Boolean!
}

type PersonInput {
    name: String!
    isMarried: Boolean!
}
```

As you can see, the input type generates the property name for `isMarried` correctly, however, I expected the same for the Person type that is being returned as the output type. Instead, it is returning as `married`.

I am using Jackson as my JSON serializer, and adding `@JsonProperty("isMarried")` to the field in the data class doesn't seem to work.

How can I enforce that the output type's `isMarried` property generates correctly to be the same as that of what is defined in the Person data class?

Any assistance in this matter will be greatly appreciated.

**Comment by kaqqao** [comment]
Answered in https://github.com/leangen/graphql-spqr/issues/462
